### DISCUSSION TITLE
Series Download

### DISCUSSION THREAD

**Discussion by Padillam5** [advplyr/audiobookshelf#826] [Ideas]
feature to download entire series, folder structure and all files in that path

exp

Series:
 book1
  cover
  book
  nfo/abs
 book2
  repeat.

so you would just click download series and it would either download direct as is but probably needs to be zipped. just a suggestion

**Comment by Junkle42** [comment]
I came here to make the same request. 
Additionally, since it would similar logic, the ability to download all the files of an audiobook that is multiple mp3 files.
